catalyst_voices/uikit_example/lib/examples/voices_indicators_example.dart

```diff
@@ -79,6 +79,14 @@ class VoicesIndicatorsExample extends StatelessWidget {
               VoicesCircularProgressIndicator(value: 0.75, showTrack: false),
             ],
           ),
+          Text('No Internet Connection Banner'),
+          Row(
+            children: [
+              Expanded(
+                child: NoInternetConnectionBanner(),
+              ),
+            ],
+          ),
```

Comment:
Row with a single child is not needed - you can safely skip it and just include the child

```suggestion
          NoInternetConnectionBanner(),
```